bin/rucio-admin

```diff
@@ -1343,6 +1344,7 @@ def get_parser():
     set_account_limits_parser.add_argument('account', action='store', help='Account name')
     set_account_limits_parser.add_argument('rse', action='store', help='RSE boolean expression')
     set_account_limits_parser.add_argument('bytes', action='store', help='Value can be specified in bytes ("10000"), with a storage unit ("10GB"), or "infinity"')
+    set_account_limits_parser.add_argument('scope', action='store', nargs='?', default='local', choices=['local', 'global'], help='Global or local limit scope. Default: "local"')
```

Comment:
I wouldn't re-use `--scope` for this, too confusing with our usual scopes. Maybe just make it a switch with `--global` and local is default?